webkit2/webkit2.web-view.lisp

```diff
@@ -39,6 +39,7 @@
    #+webkit2-mute
    ("is-muted" "gboolean")
    ("is-playing-audio" "gboolean")
+   ("page-id" "guint64")
```

Comment:
Is this related to JSC or the DOM?  Or was it just missing?

Reply:
It's related to neither of these, but I though I could use it to query WebKitWebPage (this patch was started to ressurrect Plump DOM, after all).

But now, it's just for the sake of completeness :)